hist/hist/src/TH1.cxx

```diff
@@ -873,27 +873,27 @@ Bool_t TH1::Add(const TH1 *h1, Double_t c1)
       useMerge = kFALSE;
    } catch(DifferentNumberOfBins&) {
       if (useMerge)
-         Info("Add","Attempt to add histograms with different number of bins - trying to use TH1::Merge");
+         Info("TH1::Add()","Attempt to add histograms with different number of bins - trying to use TH1::Merge");
       else {
-         Error("Add","Attempt to add histograms with different number of bins : nbins h1 = %d , nbins h2 =  %d",GetNbinsX(), h1->GetNbinsX());
+         Error("TH1::Add()","Attempt to add histograms with different number of bins : nbins h1 = %d , nbins h2 =  %d",GetNbinsX(), h1->GetNbinsX());
          return kFALSE;
       }
    } catch(DifferentAxisLimits&) {
       if (useMerge)
-         Info("Add","Attempt to add histograms with different axis limits - trying to use TH1::Merge");
+         Info("TH1::Add()","Attempt to add histograms with different axis limits - trying to use TH1::Merge");
       else
-         Warning("Add","Attempt to add histograms with different axis limits");
+         Warning("TH1::Add()","Attempt to add histograms with different axis limits");
    } catch(DifferentBinLimits&) {
       if (useMerge)
-         Info("Add","Attempt to add histograms with different bin limits - trying to use TH1::Merge");
+         Info("TH1::Add()","Attempt to add histograms with different bin limits - trying to use TH1::Merge");
       else
-         Warning("Add","Attempt to add histograms with different bin limits");
+         Warning("TH1::Add()","Attempt to add histograms with different bin limits");
    } catch(DifferentLabels&) {
       // in case of different labels -
       if (useMerge)
-         Info("Add","Attempt to add histograms with different labels - trying to use TH1::Merge");
+         Info("TH1::Add()","Attempt to add histograms with different labels - trying to use TH1::Merge");
       else
-         Info("Warning","Attempt to add histograms with different labels");
```

Comment:
Can we replace them by Warning() statements?

Reply:
`"Warning"` makes no sense here. I agree that this one should call `Warning()` not `Info()`. @lmoneta what do you think? Maybe even `Error()`?

Reply:
Docs said this should be Location. But as @linev pointed out, there are some other places with using the same structure